docs/book/how-to/popular-integrations/gcp-guide.md

```diff
@@ -175,4 +175,68 @@ If you do not want to use any of the created resources in the future, simply del
 gcloud project delete <PROJECT_ID_OR_NUMBER>
 ```
 
+## Best Practices for Using a GCP Stack with ZenML
+
+When working with a GCP stack in ZenML, consider the following best practices to optimize your workflow, enhance security, and improve cost-efficiency. These are all things you might want to do or amend in your own setup once you have tried running some pipelines on your GCP stack.
+
+### Use IAM and Least Privilege Principle
+
+Always adhere to the principle of least privilege when setting up IAM roles. Only grant the minimum permissions necessary for your ZenML pipelines to function. Regularly review and audit your IAM roles to ensure they remain appropriate and secure.
+
+### Leverage GCP Resource Labeling
+
+Implement a consistent labeling strategy for your GCP resources. To label a GCS
+bucket, for example:
+
+```shell
+gsutil label set project-labels.json gs://your-bucket-name
```

Comment:
gsutil is not the recommended CLI. use gcloud storage instead. see here: https://cloud.google.com/storage/docs/using-bucket-labels#set-label-cli

Reply:
updated now